tests/pytests/test_json_multi_text.py

```diff
@@ -637,9 +640,12 @@ def testconfigMultiTextOffsetDelta(env):
     
     # MULTI_TEXT_SLOP = 101
     env = Env(moduleArgs = 'MULTI_TEXT_SLOP 101')
```

Comment:
You should not create more then one environment on a single test, please split to multiple tests.